globus_portal_framework/templates/components/footer.html

```diff
@@ -1,3 +1,4 @@
 <!-- Footer -->
+{% load static %}
```

Comment:
I still see a the load static templatetags here

Reply:
Got it removed 👍